frontend/src/libs/components/sidebar/styles.module.scss

```diff
@@ -1,6 +1,10 @@
-.image {
-  width: 56px;
-  height: 56px;
+.logo {
+  width: 59px;
+  height: 59px;
+}
+
+.logo-img {
+  width: 59px;
 }
```

Comment:
let's add these attributes to image itself
```suggestion
```

Reply:
I'm confused can I use the global tag <img> for this one or did you mean to delete the span and give a classname to the image?

B10. No global tag styles. Only class selectors or a cascade are used to set styles.
Global styles are only available in global style files (scaffolding.css, app.css, reset.css (🥲), etc).

I will update the PR like that while you can check the question and if it needs to change I will change it again.

Reply:
I think Liza meant either of two: 
1. If it is SVG file, you can define its size right in the svg: 
<img width="983" alt="Screenshot 2023-09-04 at 12 45 40" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/13ac284c-a814-4398-92a1-dc29c5535b8d">

2. You can pass inline styling properties to the image: 
`<img src='...' width='50px />` 
Something like this

Reply:
yes, @vladyslavdobychin are right